gdcdictionary/schemas/exposure.yaml

```diff
@@ -229,11 +235,17 @@ properties:
         $ref:
           - "_terms.yaml#/not_reported/common"
 
+  exposure_duration_years:
+    $ref:
+      - "_terms.yaml#/exposure_duration_years/common"
+    type: integer
+
```

Comment:
we might be able to add a min of 0 here.

Reply:
+1

Reply:
added